manifests/plugin/dns_rfc2136.pp

```diff
@@ -0,0 +1,59 @@
+# == Class: letsencrypt::plugin::dns_rfc2136
+#
+#   This class installs and configures the Let's Encrypt dns-rfc2136 plugin.
+#   https://certbot-dns-rfc2136.readthedocs.io
+#
+# === Parameters:
+#
+# [*server*]
+#   Target DNS server.
+# [*key_name*]
+#   TSIG key name.
+# [*key_secret*]
+#   TSIG key secret.
+# [*key_algorithm*]
+#   TSIG key algorithm.
+# [*port*]
+#   Target DNS port.
+# [*manage_package*]
+#   Manage the plugin package.
+# [*config_dir*]
+#   The path to the configuration directory.
+#
+class letsencrypt::plugin::dns_rfc2136 (
+  String $server,
+  String $key_name,
+  String $key_secret,
+  String $key_algorithm = $letsencrypt::params::dns_rfc2136_algorithm,
+  String $port          = $letsencrypt::params::dns_rfc2136_port,
+  Boolean $manage_package = $letsencrypt::params::dns_rfc2136_manage_package,
+  String $config_dir      = $letsencrypt::config_dir,
```

Comment:
I assume this is always an absolute path? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/absolutepath.pp